lib/src/xml/builder.dart

```diff
@@ -298,6 +298,7 @@ class XmlBuilder {
   // Internal method to add children to the current element.
   void _insert(Object? value) {
     if (value is Function) {
+      // ignore: avoid_dynamic_calls
```

Comment:
If only a certain (or short list) of function signatures would be acceptable here, we can type-check those each.

Reply:
Yeah only `void Function()` is acceptable.

Reply:
Done